infra/README.md

```diff
@@ -0,0 +1,68 @@
+# Installation
+
+The environment can provisioned using the Bicep templates inclulded with the project and executed in a number of ways including:
+
+      1. Azure CLI
+      2. GitHub Actions
+      3. Azure DevOps
+      4. Azure Portal
+
+We will provide examples of the first two methods in this document. Manual provisioning using the Azure CLI or continuous delivery via GitHub Actions.
+
+<br/>
+
+## Deploying the Bicep templates using the Azure CLI
+
+#### :wrench: CREATING THE RESOURCE GROUP ####
+
+Before executing any of the following Bicep templates, you first need to login to your Azure account using the Azure CLI `az` command line tool:
+
+```bash
+az login
+```
+
+Then, create a resource group if it does not already exist:
+
```

Comment:
Is it worthwhile adding instructions around the following:
1. Use something like to showcase possible locations
```
az account list-locations --query "[*].name" --out tsv | sort 
```
2. Additionally, should we reference https://learn.microsoft.com/en-us/azure/cognitive-services/openai/concepts/models#gpt-3-models-1 to showcase region availability for GPT-3 and 4 models?